### DISCUSSION TITLE
Add enlarge option in display settings

### DISCUSSION THREAD

**Discussion by ghost** [bisq-network/bisq#6322] [Ideas]
Bisq application does not seem to follow accessibility settings in some Linux distributions (Im using Fedora 36 Workstation(Gnome)). Only the main system display resolution settings are being considered.

On my 16:10 2650x1600 screen this makes the text very small and difficult to read as it is not following my systems font enlarging accessibility settings. In order to make Bisq application (in my case RPM) usable I first have to adjust my display settings to 1920x1200.

Would love to see a setting in the display options for enlarging/scaling the font to make text more readable, or alternatively, follow the adjustments already made to the system in accessibility settings. All other applications on my system do not encounter this issue so hopefully it is something that could be implemented.

**Comment by satonotdead** [comment]
100% agree, I was using `--force-device-scale-factor` but I need to use an script to load and apply that to the BISQ GUI.

By default it's almost unusable on 1920x1080. Come on, all the devs are with Mac?

In that case, that's not so much decentralized :P

**Comment by ghost** [comment]
You can scale the fonts in Bisq using a command line setting described here: https://bisq.wiki/Running_on_HiDPI_screen

**Comment by ghost** [comment]
Yes, I have been using this. I would love a GUI setting to make this persist somehow across opening and closing Bisq, especially when onboarding newer users who are not familiar with the command line.
